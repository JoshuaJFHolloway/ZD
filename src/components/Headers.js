import React from 'react';
import Header from './Header';
import HeaderStyle from './styledComponents/Header.js'
import SubHeaderStyle from './styledComponents/subHeader.js'

const Headers = () => {
  return (
    <div>
      <HeaderStyle>
        <Header
          title={"Ned Ned's Cheese Emporium"}
        />
      </HeaderStyle>
      <SubHeaderStyle>
        <Header role={"img"} aria-label={"emoji"}
          title={"Today you can buy 12 blocks of cheese for Â£19.99 ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€"}
        />
      </SubHeaderStyle>
    </div>
  )
};

export default Headers;