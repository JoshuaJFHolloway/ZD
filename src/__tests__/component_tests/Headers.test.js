import React from 'react';
import { shallow } from 'enzyme';
import Headers from './../../components/Headers';
import styled from 'styled-components';

const headers = shallow(<Headers/>);

describe('Headers', () => {

  it('renders correctly', () => {
    expect(Headers).toMatchSnapshot();
  });

describe('passing props', () => {

  describe('header Component', () => {

    it('has header 1', () => {
      expect(app.find('Header').at(0).prop('title')).toMatchObject(<styled.h1>Ned Ned's Cheese Emporium</styled.h1>);
    });

    it('has header 2', () => {
      expect(app.find('Header').at(1).prop('title')).toMatchObject(<styled.h2>Today you can buy 12 blocks of cheese for Â£19.99 ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€ğŸ§€</styled.h2>);
      });
    });
  });
});