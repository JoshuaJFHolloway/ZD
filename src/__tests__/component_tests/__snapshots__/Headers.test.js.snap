// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headers renders correctly 1`] = `[Function]`;
